futaba/cogs/roles/core.py

```diff
@@ -307,7 +340,7 @@ def str_channels(channels):
     @role.command(name="pingable")
     @commands.guild_only()
     @permissions.check_mod()
-    async def role_pingable(self, ctx, role: RoleConv, *channels: TextChannelConv):
+    async def role_pingable(self, ctx, role: RoleConv, *channels: TextChannelConv, original=None):
```

Comment:
I approve of having a shared method to avoid code duplication, however I don't like it being a command. What type is `original`, and what would happen if a user passed something in there, for instance.

I would instead have this method only call the helper method, and other places it's used (e.g. `helper_role_add`) call that instead.

Reply:
A user cannot pass anything into `original`. It is a keyword argument.

Reply:
The new helper method would only be `role_pingable` with the `original` parameter. The only line in `role_pingable` would be a call to that method. I don't see a reason to move it and `original` cannot be input from discord.